### ISSUE TITLE
Analytics

### ISSUE THREAD

**Issue by lorus123** [AppCheap/roadmap#8]
Add any Analytics service to get app usage data.

- [ ] 1.Active users

- [ ] 2.Unique active users

- [x] 3.Individual Post opens

- [ ] 4.Time spend by user on each post

- [ ] 5.Scroll depth of post

- [ ] 6.Time , when user use app

- [ ] 7.Most interacted category for individual profiles (able to send notification based on this )

Maybe more...

**Comment by DrXripper** [comment]
Firebase analytics provides most of the information you asked for.

**Comment by dangtienngoc** [comment]
@DrXripper We already set up Firebase Analytics on the app, any more tasks on the app to get info like @lorus123 mention above? or don't need it.

**Comment by DrXripper** [comment]
> @DrXripper We already set up Firebase Analytics on the app, any more tasks on the app to get info like @lorus123 mentioned above? 

When will we receive 2.6.0 update?

**Comment by lorus123** [comment]
> @DrXripper We already set up Firebase Analytics on the app, any more tasks on the app to get info like @lorus123 mentioned above?

Enough for now. will try Firebase first and tell you if more type of info needed.

**Comment by dangtienngoc** [comment]
@DrXripper Yes, we just release the v2.6.0

**Comment by DrXripper** [comment]
> @DrXripper Yes, we just release the v2.6.0

Is there any documentation to use the gateway?

**Comment by dangtienngoc** [comment]
@DrXripper Yes the docs are here: https://appcheap.io/docs/app-builder-user-guide/how-to/how-to-enable-custom-checkout-flow/

But the question is not related to this issue.

**Comment by lorus123** [comment]
@dangtienngoc 

I don't see any details regarding Individual posts opens in firebase. If you are sure that firebase has this thing please share screenshot.

**Comment by lorus123** [comment]
@dangtienngoc https://pub.dev/packages/facebook_audience_network

**Comment by dangtienngoc** [comment]
Do you mean thiss?

![image](https://user-images.githubusercontent.com/4260689/157214819-294e1bf0-e8a0-4b8f-9d1d-bbdc95dcfd14.png)

**Comment by lorus123** [comment]
I can see this but i want to see posts opens.

Like /post1 
/Post2
/Post3

**Comment by dangtienngoc** [comment]
@lorus123 

Yes, I have supported display

post/id1
post/id2
...

**Comment by lorus123** [comment]
Can you share a screenshot of the data?
post/id1
post/id2

Thank you.

**Comment by dangtienngoc** [comment]
@lorus123 yes,

![image](https://user-images.githubusercontent.com/4260689/158059447-52b296f4-4f5b-4057-84e9-8812c97afdaa.png)

**Comment by lorus123** [comment]
Can it show the post title or permalink?

**Comment by dangtienngoc** [comment]
Yes, in the next version the link will include both id and slug

Ex:

https://cirilla.appcheap.io/#/post/6360/your-full-marathon-training-plan-20-weeks

https://cirilla.appcheap.io/#/product/658/the-bomber-jacket

**Comment by lorus123** [comment]
Okay

**Comment by lorus123** [comment]
@dangtienngoc 

Can i check current active users? (Users currently using app , Real time)

**Comment by lorus123** [comment]
What happened to update @dangtienngoc ?
